registry/auth/token/token.go

```diff
@@ -144,8 +147,14 @@ func (t *Token) Verify(verifyOpts VerifyOptions) error {
 
 	// Verify that the token is currently usable and not expired.
 	currentUnixTime := time.Now().Unix()
-	if !(t.Claims.NotBefore <= currentUnixTime && currentUnixTime <= t.Claims.Expiration) {
-		log.Errorf("token not to be used before %d or after %d - currently %d", t.Claims.NotBefore, t.Claims.Expiration, currentUnixTime)
+
+	if currentUnixTime >= t.Claims.Expiration+int64(Leeway.Seconds()) {
+		log.Errorf("token not to be used after %d - currently %d", t.Claims.Expiration, currentUnixTime)
+		return ErrInvalidToken
```

Comment:
@stevvooe I guess it depends how opaque to the user the token should be. But for this PR I proposing we don't change the API. I will happy to add a more explicit error message if a more explicit error message would help the user. @diogomonica @NathanMcCauley thoughts?